<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cMainWindow</class>
 <widget class="QMainWindow" name="cMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kooky</string>
  </property>
  <widget class="QWidget" name="m_lpCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="m_lpTab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="m_lpTabIngredients">
        <attribute name="title">
         <string>Ingredients</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTreeView" name="m_lpIngredientsList">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_lpTabReceips">
        <attribute name="title">
         <string>Receips</string>
        </attribute>
       </widget>
      </widget>
          <widget class="cMdiArea" name="m_lpMDIArea">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_lpMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_lpMenu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_lpMenu_File_Exit"/>
   </widget>
   <widget class="QMenu" name="m_lpMenuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
    <widget class="QMenu" name="m_lpMenu_Plugins_Import">
     <property name="title">
      <string>&amp;Import</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="m_lpMenu_Plugins_Export">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="m_lpMenu_Plugins_Database">
     <property name="title">
      <string>Database</string>
     </property>
    </widget>
    <addaction name="m_lpMenu_Plugins_Import"/>
    <addaction name="m_lpMenu_Plugins_Export"/>
    <addaction name="m_lpMenu_Plugins_Database"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="m_lpMenu_Tools_Options"/>
   </widget>
   <addaction name="m_lpMenu_File"/>
   <addaction name="m_lpMenuPlugins"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QToolBar" name="m_lpMainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_lpStatusBar"/>
  <action name="m_lpMenu_File_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="m_lpMenu_Tools_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cMdiArea</class>
   <extends>QFrame</extends>
   <header>cmdiarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
